### PR TITLE
rename vc `*_timestamp` fields

### PR DESCRIPTION
**Pull request by kziemianek**
Fields holding datetime should be represented by ISO-8601 datetime format instead of numerical timestamp.

renames created_timestamp -> created
renames issuanceTimestamp -> issuanceDate
removes expirationTimestamp

### CODE REVIEW

File: tee-worker/litentry/core/credentials/src/lib.rs [hunk 1]
```diff
@@ -186,7 +186,7 @@ pub struct Proof {
 impl Proof {
 	pub fn new(sig: &Vec<u8>, issuer: &AccountId) -> Self {
 		Proof {
-			created_timestamp: now_as_millis(),
+			created: now_as_millis(),
```
Comment (arieljbirnbaum) [inline tee-worker/litentry/core/credentials/src/lib.rs]: If the goal is to align with the spec, shouldn't `created` be in ISO-8601 format instead?

Comment (kziemianek) [reply tee-worker/litentry/core/credentials/src/lib.rs]: Yes, we will change all datetimes representations from timestamps to ISO-8601

Comment (arieljbirnbaum) [reply tee-worker/litentry/core/credentials/src/lib.rs]: Here or in a different PR?

Comment (kziemianek) [reply tee-worker/litentry/core/credentials/src/lib.rs]: I'll add it here to make this change complete.

File: tee-worker/core-primitives/time-utils/Cargo.toml [hunk 2]
```diff
@@ -5,12 +5,18 @@ authors = ["Integritee AG <hello@integritee.network>"]
 edition = "2021"
 
 [dependencies]
+chrono = { version = "0.4.19", features = ["alloc"], optional = true }
+
+
+chrono_sgx = { package = "chrono", git = "https://github.com/mesalock-linux/chrono-sgx", optional = true }
```
Comment (arieljbirnbaum) [inline tee-worker/core-primitives/time-utils/Cargo.toml]: Two (noob) questions: 1. they're marked as `optional=true` to allow the `#[cfg]` below to choose one or the other? and 2. do the empty lines matter? :D

File: tee-worker/core-primitives/time-utils/src/lib.rs [hunk 3]
```diff
@@ -36,6 +43,11 @@ pub fn now_as_millis() -> u64 {
 	duration_now().as_millis() as u64
 }
 
+pub fn now_as_iso8601() -> String {
+	let date_time: DateTime<Utc> = SystemTime::now().into();
```
Comment (arieljbirnbaum) [inline tee-worker/core-primitives/time-utils/src/lib.rs:47]: The timezone is determined by _the type_? I... do not hate this :star_struck:

File: tee-worker/core-primitives/time-utils/Cargo.toml (hunk 2 above)
Comment (kziemianek) [reply tee-worker/core-primitives/time-utils/Cargo.toml]: I'll remove this redundant new line.

Regarding optional dependencies: yes we want to enable them conditionally (std/sgx).
https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

**Review by arieljbirnbaum — APPROVED** [review]
lgtm

**Review by BillyWooo — APPROVED** [review]

**Comment by Kailai-Wang** [comment]
Would you like to change `issuanceTimestamp` to ` issuanceDate` as well to be more w3c compliant? This is a field outside of `proof` section

**Comment by kziemianek** [comment]
> Would you like to change `issuanceTimestamp` to ` issuanceDate` as well to be more w3c compliant? This is a field outside of `proof` section

Yes, i'll add more changes to this PR.

**Pull request merged**